docs/application/native/tutorials/details/tizen-apis.md

```diff
@@ -195,20 +195,20 @@ different API modules to decide which APIs to use.
-| [Application  Framework](../../../../org.tizen.native.wearable.apireference/group__CAPI__APPLICATION__FRAMEWORK.html) | [Service  Application](../../../../org.tizen.native.wearable.apireference/group__CAPI__SERVICE__APP__MODULE.html) | [Handles  Tizen service application (non-UI application) state changes and  system events. It is also used to start and exit the main event loop of  service applications.](../../guides/app-management/service-app.md) | The  Application Framework API contains submodule APIs for application  development. The submodule APIs enable application life-cycle management,  usage of functionality that is exported by other applications, and access to  a user's application preferences. <br>Guides:<br> - [Alarms and Scheduling](../../guides/alarm/alarm-cover.md)<br> - [Application Management](../../guides/app-management/app-management-cover.md)<br> - [Notifications and Content Sharing](../../guides/notification/notification-cover.md)   |
+| [Application  Framework](../../../../org.tizen.native.wearable.apireference/group__CAPI__APPLICATION__FRAMEWORK.html) | [Service  Application](../../../../org.tizen.native.wearable.apireference/group__CAPI__SERVICE__APP__MODULE.html) | Handles  Tizen [service application](../../guides/app-management/service-app.md) (non-UI application) state changes and  system events. It is also used to start and exit the main event loop of  service applications. | The  Application Framework API contains submodule APIs for application  development. The submodule APIs enable application life-cycle management,  usage of functionality that is exported by other applications, and access to  a user's application preferences. <br>Guides:<br> - [Alarms and Scheduling](../../guides/alarm/alarm-cover.md)<br> - [Application Management](../../guides/app-management/app-management-cover.md)<br> - [Notifications and Content Sharing](../../guides/notification/notification-cover.md)   |
```

Comment:
alarm-cover.md should be changed into overview.md.
app-management-cover.md should be changed into index.md.
notification-cover.md should be changed into overview.md.

The whole links should be checked, I think..

Reply:
All contents were moved to native/guides/index.md so I delete it.